tests/upgrades/test_errata.py

```diff
@@ -434,15 +385,14 @@ def test_post_scenario_generate_errata_with_previous_version_katello_agent_clien
         for errata in settings.repos.yum_9.errata:
             host.errata_apply(data={'errata_ids': [errata]})
 
-        for package in FAKE_9_YUM_UPDATED_PACKAGES:
-            install_or_update_package(client_hostname=client_container_id, package=package)
+        rhel_client.execute(f'yum -y install {" ".join(constants.FAKE_9_YUM_UPDATED_PACKAGES)}')
+        # Do we need to install the packages on the client?
+        # The yum_9.errata(s) should already be installed
```

Comment:
I believe so! `host.errata_apply` should have done it. But does the `ettings.repos.yum_9.errata` is same as `FAKE_9_YUM_UPDATED_PACKAGES` ?

IF they are same just same then remove, if not just verify the errata from `settings.repos.yum_9.errata` is applied/installed.

Reply:
I have added a `rpm -q` check to verify that all packages were installed successfully.

Reply:
I can also do the following assertion:
```python
errata_check = rhel_client.execute('dnf updateinfo --list')
for errata in settings.repos.yum_9.errata:
assert errata not in errata_check.stdout 
```